changes.txt

```diff
@@ -0,0 +1,84 @@
+diff --git a/app/controllers/contacts_controller.rb b/app/controllers/contacts_controller.rb
```

Comment:
lol what is this file?

Reply:
woops this was me testing a git command my bad ill delete